Drive/rover_drive_system.hpp

```diff
@@ -351,23 +363,16 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
     units::angle::degree_t right_wheel_angle =
         angle;  // Needs to be set by server
     double x_angle = static_cast<double>(right_wheel_angle);
-    units::angle::degree_t left_wheel_angle(static_cast<double>(
-        0.392 + 0.744 * x_angle + -0.0187 * pow(x_angle, 2) +
-        1.84E-04 * pow(x_angle, 3)));
 
-    units::angle::degree_t back_wheel_angle(static_cast<double>(
-        -0.378 + -1.79 * x_angle + 0.0366 * pow(x_angle, 2) +
-        -3.24E-04 * pow(x_angle, 3)));
+    double left_angle = calculateLeftTurnAngle(x_angle);
+    double back_angle = calculateBackTurnAngle(x_angle);
+
+    units::angle::degree_t left_wheel_angle(left_angle);
+    units::angle::degree_t back_wheel_angle(back_angle);
```

Comment:
```suggestion
    units::angle::degree_t left_wheel_angle(calculateLeftTurnAngle());
    units::angle::degree_t back_wheel_angle(calculateBackTurnAngle());
```
This could remove need for local variables. Might look cleaner...